llama-index-integrations/vector_stores/llama-index-vector-stores-duckdb/llama_index/vector_stores/duckdb/base.py

```diff
@@ -95,10 +95,10 @@ class DuckDBVectorStore(BasePydanticVectorStore):
 
     def __init__(
         self,
+        embed_dim: int,
         database_name: Optional[str] = ":memory:",
         table_name: Optional[str] = "documents",
         # schema_name: Optional[str] = "main",
-        embed_dim: Optional[int] = 1536,
```

Comment:
This is a breaking interface change. Curious what the motivation here is?

Reply:
added it back